<launch>

	<param name="vad" value="webrtc" />
	<param name="output_speaker" value="pepper" />

	<param name="input" value="topic" />
	<param name="input_topic" value="naoqi_microphone/audio_raw" />
	<param name="input_topic_msg" value="AudioBuffer" />
	<include file="$(find naoqi_sensors_py)/launch/microphone.launch"/>

	<node name="tts" pkg="qt-dialog" type="tts_node.py" output="screen" cwd="node"/>
	<node name="stt" pkg="qt-dialog" type="stt_node.py" output="screen" cwd="node"/>
	<node name="nlp" pkg="qt-dialog" type="nlp_node.py" output="screen" cwd="node"/>
	<node name="vad" pkg="qt-dialog" type="vad_node.py" output="screen" cwd="node"/>
	<node name="dialog" pkg="qt-dialog" type="dialog_node.py" output="screen" cwd="node"/>

</launch>
